test-dsl/src/main/groovy/com/smartbear/readyapi4j/dsl/assertions/ResponseSlaAssertionDelegate.groovy

```diff
@@ -0,0 +1,42 @@
+package com.smartbear.readyapi4j.dsl.assertions
+
+import com.smartbear.readyapi4j.assertions.AssertionBuilder
+import com.smartbear.readyapi4j.assertions.Assertions
+
+
+class ResponseSlaAssertionDelegate {
+    private int time
+    private List<AssertionBuilder> assertionBuilders
+
+    ResponseSlaAssertionDelegate(int time, List<AssertionBuilder> assertionBuilders) {
+        this.time = time
+        this.assertionBuilders = assertionBuilders
+    }
+
+    int getMilliseconds() {
```

Comment:
Maybe we should have a getMs() method too, since ms is such a common abbreviation?